assets/js/messages.json

```diff
@@ -326,6 +326,14 @@
                 "fillname": []
             }
         ],
+        [
+            {
+                "project": ["mc"],
+                "name": "Provide seed and coordinates",
+                "message": "*Thank you for your report!*\nHowever, this issue has been temporarily closed as {color:#FF5722}*Awaiting Response*{color}.\n\nTo be able to reproduce this, we need to know some information about the the world you are playing in.\nPlease provide the following information as text, attaching a screenshot showing them is not enough:\n- World Type\n- Seed: Can be obtained by using the {{/seed}} command\n- Coordinates: Can be copied to clipboard by pressing {{F3}} + {{C}}\n\nOnce the requested information has been delivered, the report will be reopened automatically.\n\n%quick_links%",
```

Comment:
- World Type: {{Default}} if you haven't chosen one. If you used a customizable world type, also include the generation preset.

Reply:
What about this, or it that overkill?
```
*Thank you for your report!*\nHowever, this issue has been temporarily closed as {color:#FF5722}*Awaiting Response*{color}.\n\nTo be able to reproduce this, we need to know some information about the the world you are playing in.\nPlease provide the following information as text, attaching a screenshot showing them is not enough:\n- World Type:\n-- "Default" if you have not chosen one when creating the world.\n-- For "Superflat": Additionally provide the generation preset.\n-- For "Buffet": Additionally provide the generator type and biome you have selected.\n- Additional generation settings (in case you changed them): E.g. "Generate Structures", "Bonus Chest", ...\n- Seed: Can be obtained by using the {{/seed}} command and clicking on the message in chat\n- Coordinates: Can be copied to clipboard by pressing {{F3}} + {{C}}\n\nOnce the requested information has been delivered, the report will be reopened automatically.\n\n
```
![Jira text preview](https://user-images.githubusercontent.com/11685886/79580813-de06f080-80c9-11ea-9856-e136bd94239a.png)

Reply:
I dunno, I don't think it's overkill, and I like more verbosity in these messages anyways, just to avoid confusion and also guide people who don't know some of the technical details.
This one looks good to me.